packages/table/src/react/__specs__/index.spec.js

```diff
@@ -6,8 +6,9 @@ import Table from '../index.js'
 describe('columnHeader', () => {
   test('sorted=false onClick called with asc', () => {
     return new Promise(resolve => {
-      const onClick = sort => {
+      const onClick = (sort, evt) => {
         expect(sort).toBe(Table.sorts.asc)
+        expect(evt).toBeDefined()
         resolve()
       }
```

Comment:
evt as first argument would be more consistent with react general practices...but that's a breaking change. would could break now or later 🤷

Reply:
Yeah, I want to be consistent one way or the other.  

We have a mix today:

After:
```
drawer/src/react/index.js                                                                                      
72:    onToggle && onToggle(nextOpen, evt)                                                                     
                                                                                                               
starrating/src/react/index.js                                                                                  
34:    props.onChange(evt.target.value, evt)                                                                   
37:  function handleStarClicked(index, evt) {                                                                  
39:    props.onChange(index + 1, evt)                                                                          
```

Before:
```
checkbox/src/react/index.js                                                                                    
59:      onCheck(evt, !props.checked, props.value, props.name)                                                 
                                                                                                               
dropdown/src/react/index.js                                                                                    
146:  function handleMenuChange(evt, value, label) {                                                           
150:      allProps.onChange(evt, value, label)                                                                 
                                                                                                               
typeahead/src/react/index.js                                                                                   
84:  const handleChange = (evt, nextValue) => {                                                                
88:    if (onChange) onChange(evt, nextValue)                                                                  
98:    handleChange(evt, nextValue)                                                                            
101:  const handleSuggestionMenuChange = (evt, nextValue) => {                                                 
103:    handleChange(evt, nextValue)                                                                           
                                                                                                               
viewtoggle/src/react/index.js                                                                                  
48:  function handleSelect(evt, index) {                                                                       
51:    if (evt && isFunction(onSelect)) onSelect(evt, index)                                                   
119:    props._onSelect(evt, props._i) 
```

What's the React general practice?  Where could I observe it?